src/vs/workbench/contrib/debug/browser/media/debugViewlet.css

```diff
@@ -246,10 +246,6 @@
 	opacity: 0.35;
 }
 
-.monaco-workbench .debug-viewlet .monaco-list-row .expression {
-	white-space: pre;
```

Comment:
@isidorn the variables view already preserved whitespaces here, I just delete this to avoid redundant css rules and moved it to `debug.contribution.css`. If I change it to `nowrap` it would be the same issue as it would be an incorrect/misleading value for strings containing whitespaces.